cmd/ketch/platform_list_test.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/shipa-corp/ketch/internal/api/v1beta1"
+)
+
+func TestPlatformList(t *testing.T) {
+	expected := `+------+-----------------------+-------------+
```

Comment:
should we start using it? `app list` and `pool list` use a style similar to kubectl. This type of table data is hard to parse and I'm not sure how to consume this kind of output in scripts